### PR TITLE
DIV-3773 Added helm chart config

### PR DESCRIPTION
**Pull request by tchow8**
# Description

Added chart config

Fixes # (issue)

[DIV-3773](https://tools.hmcts.net/jira/browse/DIV-3773)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by tchow8** [comment]
@dtaylor7 @hosainnet Reviews please. Had to close the old PR because it got corrupted in Kube.

**Comment by tchow8** [comment]
Auto whitelist proven by https://github.com/hmcts/div-decree-nisi-frontend/pull/323

File: charts/div-dn/values.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+nodejs:
+    applicationPort: 3000
+    environment:
+        CCD_DIGITAL_COURTS: '["serviceCentre", "eastMidlands", "westMidlands", "southWest", "northWest"]'
+        GOOGLE_ANALYTICS_TRACKING_URL: "http://www.google-analytics.com/collect"
+        EVIDENCE_MANAGEMENT_CLIENT_API_UPLOAD_ENDPOINT: "/emclientapi/version/1/upload"
+        HPKP_MAX_AGE: "86400"
+        HPKP_SHAS: "Naw+prhcXSIkbtYJ0t7vAD+Fc92DWL9UZevVfWBvids=,klO23nT2ehFDXCfx3eHTDRESMz3asj1muO+4aIdjiuY=,grX4Ta9HpZx6tSHkmCrvpApTQGo67CYDnvprLg5yRME="
+        IDAM_PROXY: "http://proxyout.reform.hmcts.net:8080"
```
Comment (hosainnet) [inline charts/div-dn/values.yaml]: Not sure why the app should know about `IDAM_PROXY` at all..

Comment (hosainnet) [reply charts/div-dn/values.yaml]: I see it was introduced back when we needed to use proxy for healthcheck, but was left here after that was taken out

**Pull request merged**